### ISSUE TITLE
Dashboard shows errors when creating storage-vpool

### ISSUE THREAD

**Issue by bennypi** [EMCECS/ECS-CommunityEdition#197]
I'm trying to install v.3.0.0.1 as a single node deployment. When the fix from issue  https://github.com/EMCECS/ECS-CommunityEdition/issues/196 is applied, the script for step1 is running without errors. If needed, I can attach the complete console output from step1. 

I can also change the root password and upload the lincense file without any errors. After I click on save as mentioned in this section https://github.com/EMCECS/ECS-CommunityEdition/blob/master/Documentation/ECS-UI-Web-Interface.md#create-storage-vpool and wait 20 minutes, the dashboard gets really slow. At first it only shows those rotating loading arrows, then the following errors are shown:

![dashboard_emc](https://cloud.githubusercontent.com/assets/2631090/24456747/4da777c0-1494-11e7-8be0-06fdf0501860.png)

/var/log/vipr/emc-viprobject/ssm.log shows the following errors:
```
2017-03-29T13:01:33,399 [ZkConnection-38]  INFO  ZkConnection.java (line 216) check for changes in /data/vnest/vnest-main/configuration/coord-client
2017-03-29T13:01:33,954 [TaskScheduler-SSManager-ParallelExecutor-077-0ad2f80f:15b1a06464c:23:1c]  INFO  RoutingClientBase.java (line 753) Retry on failure - dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list, starting attempt# 5 requestId 0ad2f80f:15b1a0657bd:db:c, commandType REQUEST_BLOCK_REALLOCATION, to 10.210.248.15:1095
2017-03-29T13:01:33,955 [TaskScheduler-SSManager-ParallelExecutor-077-0ad2f80f:15b1a06464c:23:1c]  WARN  RoutingClientBase.java (line 640) Request failed, server 10.210.248.15:1095, key [CoS: urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690, type: SS, keyHash: 5219, level: 1], requestId 0ad2f80f:15b1a0657bd:db:c, Process SSM, com.emc.storageos.data.object.exception.ObjectControllerException: dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list
2017-03-29T13:01:34,555 [ZoneScanner-000]  INFO  RoutingClientBase.java (line 753) Retry on failure - RT DT from urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690 is not ready yet, please come back later, starting attempt# 7 requestId 0ad2f80f:15b1a0657bd:5:1f5, commandType REQUEST_RG_GET, to null
2017-03-29T13:01:34,557 [ZoneScanner-000]  INFO  RoutingClientBase.java (line 858) directory for key [CoS: urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690, type: RT, keyHash: 3724, level: 0] is not ready yet
2017-03-29T13:01:34,864 [TaskScheduler-SSManager-ParallelExecutor-179-0ad2f80f:15b1a06464c:23:1c]  INFO  RoutingClientBase.java (line 753) Retry on failure - dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list, starting attempt# 13 requestId 0ad2f80f:15b1a0657bd:155:19, commandType REQUEST_BLOCK_REALLOCATION, to 10.210.248.15:1095
2017-03-29T13:01:34,864 [TaskScheduler-SSManager-ParallelExecutor-179-0ad2f80f:15b1a06464c:23:1c]  WARN  RoutingClientBase.java (line 640) Request failed, server 10.210.248.15:1095, key [CoS: urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690, type: SS, keyHash: 5219, level: 1], requestId 0ad2f80f:15b1a0657bd:155:19, Process SSM, com.emc.storageos.data.object.exception.ObjectControllerException: dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list
2017-03-29T13:01:34,865 [TaskScheduler-SSManager-ParallelExecutor-179-0ad2f80f:15b1a06464c:23:1c] ERROR  RoutingClientBase.java (line 523) failed to send request 0ad2f80f:15b1a0657bd:155:19 to server 10.210.248.15, remaining time millis -33017
2017-03-29T13:01:34,865 [TaskScheduler-SSManager-ParallelExecutor-179-0ad2f80f:15b1a06464c:23:1c] ERROR  RoutingClientBase.java (line 1030) sendAndWaitForResponseWithRetryAndUpdate hit an exception for request [0ad2f80f:15b1a0657bd:155:19, 10.210.248.15, 1095] server com.emc.storageos.data.object.exception.ObjectControllerException: failed to send request 0ad2f80f:15b1a0657bd:155:19, serverIp 10.210.248.15, remaining time millis -33017
2017-03-29T13:01:34,865 [TaskScheduler-SSManager-ParallelExecutor-179-0ad2f80f:15b1a06464c:23:1c] ERROR  InterSSManagerCommunicationClient.java (line 131) reallocation exception
com.emc.storageos.data.object.exception.ObjectControllerException: failed to send request 0ad2f80f:15b1a0657bd:155:19, serverIp 10.210.248.15, remaining time millis -33017
        at com.emc.storageos.data.object.directoryclient.RoutingClientBase.getAndCheckTimeRemaining(RoutingClientBase.java:536)
        at com.emc.storageos.data.object.directoryclient.RoutingClientBase.sendAndWaitForResponseWithRetry(RoutingClientBase.java:716)
        at com.emc.storageos.data.object.directoryclient.RoutingClientBase.sendAndWaitForResponseWithRetryAndUpdate(RoutingClientBase.java:965)
        at com.emc.storageos.data.ssmanager.ipc.InterSSManagerCommunicationClient.communicate(InterSSManagerCommunicationClient.java:354)
        at com.emc.storageos.data.ssmanager.ipc.InterSSManagerCommunicationClient.reallocateBlocks(InterSSManagerCommunicationClient.java:164)
        at com.emc.storageos.data.ssmanager.ipc.InterSSManagerCommunicationClient.reallocateBlocks(InterSSManagerCommunicationClient.java:109)
        at com.emc.storageos.data.ssmanager.ipc.InterSSManagerCommunicationClient.reallocateBlocks(InterSSManagerCommunicationClient.java:57)
        at com.emc.storageos.data.ssmanager.sstable.SSTable.allocateBlockThroughReallocation(SSTable.java:585)
        at com.emc.storageos.data.ssmanager.sstable.SSTable.allocateBlock(SSTable.java:305)
        at com.emc.storageos.data.ssmanager.SSManagerRequestProcessor.processAllocateBlockRequest(SSManagerRequestProcessor.java:453)
        at com.emc.storageos.data.ssmanager.SSManagerRequestProcessor.handleRequest(SSManagerRequestProcessor.java:149)
        at com.emc.storageos.data.object.communicator.AbstractRequestProcessor.handleRemoteRequest(AbstractRequestProcessor.java:212)
        at com.emc.storageos.data.object.communicator.impl.RemoteOperationProcessor$RequestProcessor.run(RemoteOperationProcessor.java:288)
        at com.emc.storageos.data.object.communicator.impl.RemoteOperationProcessor$RequestProcessor.call(RemoteOperationProcessor.java:266)
        at com.emc.storageos.data.object.communicator.impl.RemoteOperationProcessor$RequestProcessor.call(RemoteOperationProcessor.java:256)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at com.emc.storageos.data.object.impl.BlockingThreadPoolExecutor$2.run(BlockingThreadPoolExecutor.java:98)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
2017-03-29T13:01:34,866 [TaskScheduler-SSManager-ParallelExecutor-179-0ad2f80f:15b1a06464c:23:1c]  INFO  InterSSManagerCommunicationClient.java (line 135) Could not get any blocks though reallocation for SS 10.210.248.15
2017-03-29T13:01:34,866 [TaskScheduler-SSManager-ParallelExecutor-179-0ad2f80f:15b1a06464c:23:1c]  INFO  SSTable.java (line 590) Could not allocate blocks through reallocation in level 2 DT urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_2: for SS 10.210.248.15 , chunk 17d33b20-1c30-4234-9c01-5e8d7ac2aafe and request 0ad2f80f:15b1a065f21:1a:16
2017-03-29T13:01:35,025 [SSConfigurationTracker-000]  INFO  SSPinger.java (line 70) Staring PING request to SS 10.210.248.15 with seq 1407
2017-03-29T13:01:35,027 [SSConfigurationTracker-000]  WARN  SSPinger.java (line 117) Partition:e9f8168a-c5a1-44d0-a8ea-adff1d7ea3f6 with state:SUSPECT has no block bins. SS 10.210.248.15, seq 1407
2017-03-29T13:01:35,027 [SSConfigurationTracker-000]  INFO  SSRecordHolder.java (line 146) Set size 0, SS 10.210.248.15
2017-03-29T13:01:35,028 [SSConfigurationTracker-000]  INFO  SSPinger.java (line 92) Completed PING request to SS 10.210.248.15 with seq 1407
2017-03-29T13:01:36,535 [TaskScheduler-SSManager-ParallelExecutor-105-0ad2f80f:15b1a06464c:23:1c]  INFO  RoutingClientBase.java (line 753) Retry on failure - dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list, starting attempt# 5 requestId 0ad2f80f:15b1a0657bd:175:e, commandType REQUEST_BLOCK_REALLOCATION, to null
2017-03-29T13:01:36,535 [TaskScheduler-SSManager-ParallelExecutor-105-0ad2f80f:15b1a06464c:23:1c]  INFO  BlackList.java (line 59) removed urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: from dt black list ; expire 4207903738043 removeOrNot true nanoTime 4208886630814
2017-03-29T13:01:36,535 [TaskScheduler-SSManager-ParallelExecutor-105-0ad2f80f:15b1a06464c:23:1c]  INFO  BufferedFileWriterSharedChunkPrefetcher.java (line 570) try to trigger chunk creation for DT urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1:9bc29876-9185-4ee4-89f8-4eac722b1f74
2017-03-29T13:01:36,536 [TaskScheduler-SSManager-ParallelExecutor-105-0ad2f80f:15b1a06464c:23:1c]  WARN  AbstractRequestProcessor.java (line 217) failure processing remote request ID 0ad2f80f:15b1a0657bd:175:e com.emc.storageos.data.object.exception.ObjectControllerException: is readOnly, chunk or JR is not created yet, urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1:
2017-03-29T13:01:36,536 [TaskScheduler-SSManager-ParallelExecutor-105-0ad2f80f:15b1a06464c:23:1c]  INFO  AbstractRequestProcessor.java (line 305) Cannot utilize directory table for key hash 5219 due to error is readOnly, chunk or JR is not created yet, urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1:: request from 10.210.248.15 - 0ad2f80f:15b1a0657bd:175:e
2017-03-29T13:01:36,536 [TaskScheduler-SSManager-ParallelExecutor-105-0ad2f80f:15b1a06464c:23:1c]  WARN  RoutingClientBase.java (line 640) Request failed, server 10.210.248.15:1095, key [CoS: urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690, type: SS, keyHash: 5219, level: 1], requestId 0ad2f80f:15b1a0657bd:175:e, Process SSM, com.emc.storageos.data.object.exception.ObjectControllerException: DT not initialized, request 0ad2f80f:15b1a0657bd:175:e, ip 10.210.248.15:1095
2017-03-29T13:01:36,536 [TaskScheduler-SSManager-ParallelExecutor-105-0ad2f80f:15b1a06464c:23:1c]  INFO  BlackList.java (line 46) added urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: to dt black list ; expire 4213888103237 prev null
2017-03-29T13:01:37,442 [TaskScheduler-SSManager-ParallelExecutor-083-0ad2f80f:15b1a06464c:23:1c]  INFO  RoutingClientBase.java (line 753) Retry on failure - dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list, starting attempt# 13 requestId 0ad2f80f:15b1a0657bd:c5:15, commandType REQUEST_BLOCK_REALLOCATION, to 10.210.248.15:1095
2017-03-29T13:01:37,442 [TaskScheduler-SSManager-ParallelExecutor-083-0ad2f80f:15b1a06464c:23:1c]  WARN  RoutingClientBase.java (line 640) Request failed, server 10.210.248.15:1095, key [CoS: urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690, type: SS, keyHash: 5219, level: 1], requestId 0ad2f80f:15b1a0657bd:c5:15, Process SSM, com.emc.storageos.data.object.exception.ObjectControllerException: dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list
2017-03-29T13:01:37,442 [TaskScheduler-SSManager-ParallelExecutor-083-0ad2f80f:15b1a06464c:23:1c] ERROR  RoutingClientBase.java (line 523) failed to send request 0ad2f80f:15b1a0657bd:c5:15 to server 10.210.248.15, remaining time millis -33015
2017-03-29T13:01:37,443 [TaskScheduler-SSManager-ParallelExecutor-083-0ad2f80f:15b1a06464c:23:1c] ERROR  RoutingClientBase.java (line 1030) sendAndWaitForResponseWithRetryAndUpdate hit an exception for request [0ad2f80f:15b1a0657bd:c5:15, 10.210.248.15, 1095] server com.emc.storageos.data.object.exception.ObjectControllerException: failed to send request 0ad2f80f:15b1a0657bd:c5:15, serverIp 10.210.248.15, remaining time millis -33015
2017-03-29T13:01:37,443 [TaskScheduler-SSManager-ParallelExecutor-083-0ad2f80f:15b1a06464c:23:1c] ERROR  InterSSManagerCommunicationClient.java (line 131) reallocation exception
com.emc.storageos.data.object.exception.ObjectControllerException: failed to send request 0ad2f80f:15b1a0657bd:c5:15, serverIp 10.210.248.15, remaining time millis -33015
        at com.emc.storageos.data.object.directoryclient.RoutingClientBase.getAndCheckTimeRemaining(RoutingClientBase.java:536)
        at com.emc.storageos.data.object.directoryclient.RoutingClientBase.sendAndWaitForResponseWithRetry(RoutingClientBase.java:716)
        at com.emc.storageos.data.object.directoryclient.RoutingClientBase.sendAndWaitForResponseWithRetryAndUpdate(RoutingClientBase.java:965)
        at com.emc.storageos.data.ssmanager.ipc.InterSSManagerCommunicationClient.communicate(InterSSManagerCommunicationClient.java:354)
        at com.emc.storageos.data.ssmanager.ipc.InterSSManagerCommunicationClient.reallocateBlocks(InterSSManagerCommunicationClient.java:164)
        at com.emc.storageos.data.ssmanager.ipc.InterSSManagerCommunicationClient.reallocateBlocks(InterSSManagerCommunicationClient.java:109)
        at com.emc.storageos.data.ssmanager.ipc.InterSSManagerCommunicationClient.reallocateBlocks(InterSSManagerCommunicationClient.java:57)
        at com.emc.storageos.data.ssmanager.sstable.SSTable.allocateBlockThroughReallocation(SSTable.java:585)
        at com.emc.storageos.data.ssmanager.sstable.SSTable.allocateBlock(SSTable.java:305)
        at com.emc.storageos.data.ssmanager.SSManagerRequestProcessor.processAllocateBlockRequest(SSManagerRequestProcessor.java:453)
        at com.emc.storageos.data.ssmanager.SSManagerRequestProcessor.handleRequest(SSManagerRequestProcessor.java:149)
        at com.emc.storageos.data.object.communicator.AbstractRequestProcessor.handleRemoteRequest(AbstractRequestProcessor.java:212)
        at com.emc.storageos.data.object.communicator.impl.RemoteOperationProcessor$RequestProcessor.run(RemoteOperationProcessor.java:288)
        at com.emc.storageos.data.object.communicator.impl.RemoteOperationProcessor$RequestProcessor.call(RemoteOperationProcessor.java:266)
        at com.emc.storageos.data.object.communicator.impl.RemoteOperationProcessor$RequestProcessor.call(RemoteOperationProcessor.java:256)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at com.emc.storageos.data.object.impl.BlockingThreadPoolExecutor$2.run(BlockingThreadPoolExecutor.java:98)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
2017-03-29T13:01:37,443 [TaskScheduler-SSManager-ParallelExecutor-083-0ad2f80f:15b1a06464c:23:1c]  INFO  InterSSManagerCommunicationClient.java (line 135) Could not get any blocks though reallocation for SS 10.210.248.15
2017-03-29T13:01:37,443 [TaskScheduler-SSManager-ParallelExecutor-083-0ad2f80f:15b1a06464c:23:1c]  INFO  SSTable.java (line 590) Could not allocate blocks through reallocation in level 2 DT urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_2: for SS 10.210.248.15 , chunk 358c2385-b423-44c6-b080-fb275c202af4 and request 0ad2f80f:15b1a065f21:146:12
2017-03-29T13:01:39,552 [TaskScheduler-SSManager-ParallelExecutor-152-0ad2f80f:15b1a06464c:23:1c]  INFO  RoutingClientBase.java (line 753) Retry on failure - DT not initialized, request 0ad2f80f:15b1a0657bd:122:16, ip 10.210.248.15:1095, starting attempt# 3 requestId 0ad2f80f:15b1a0657bd:122:16, commandType REQUEST_BLOCK_REALLOCATION, to 10.210.248.15:1095
2017-03-29T13:01:39,553 [TaskScheduler-SSManager-ParallelExecutor-152-0ad2f80f:15b1a06464c:23:1c]  WARN  RoutingClientBase.java (line 640) Request failed, server 10.210.248.15:1095, key [CoS: urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690, type: SS, keyHash: 5219, level: 1], requestId 0ad2f80f:15b1a0657bd:122:16, Process SSM, com.emc.storageos.data.object.exception.ObjectControllerException: dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list
2017-03-29T13:01:39,591 [TaskScheduler-SSManager-DefaultScheduler-002]  INFO  TaskScheduler.java (line 474) Thread stats - task scheduler DEFAULT_BACKGROUND_OPERATION: the active thread 0, core pool size 75, pool size 75, queued tasks 288
2017-03-29T13:01:39,592 [TaskScheduler-SSManager-DefaultScheduler-002]  INFO  TaskScheduler.java (line 474) Thread stats - task scheduler CHUNK_PREFETCH_HINT_REFRESHER: the active thread 0, core pool size 10, pool size 10, queued tasks 3
2017-03-29T13:01:39,592 [TaskScheduler-SSManager-DefaultScheduler-002]  INFO  TaskScheduler.java (line 474) Thread stats - task scheduler BPLUSTREE_SCAN: the active thread 0, core pool size 1, pool size 1, queued tasks 32
2017-03-29T13:01:39,592 [TaskScheduler-SSManager-DefaultScheduler-002]  INFO  TaskScheduler.java (line 474) Thread stats - task scheduler UPDATE_REGIONS: the active thread 0, core pool size 5, pool size 5, queued tasks 32
2017-03-29T13:01:39,592 [TaskScheduler-SSManager-DefaultScheduler-002]  INFO  TaskScheduler.java (line 474) Thread stats - task scheduler INDEX_STORE_SEALER: the active thread 0, core pool size 5, pool size 5, queued tasks 32
2017-03-29T13:01:39,592 [TaskScheduler-SSManager-DefaultScheduler-002]  INFO  TaskScheduler.java (line 486) Thread stats: shared scheduler executor: active thread 1,pool size 10, queued tasks 2; shared parallel executor: active thread 19,pool size 200, queued tasks 0
2017-03-29T13:01:39,592 [TaskScheduler-SSManager-DefaultScheduler-002]  INFO  TaskScheduler.java (line 496) Thread stats - total in thread pools: the active thread 20, pool size 306, queued tasks 389
2017-03-29T13:01:39,592 [TaskScheduler-SSManager-DefaultScheduler-002]  INFO  TaskScheduler.java (line 507) Thread stats - total threads in application 550, total tasks 64
2017-03-29T13:01:39,975 [TaskScheduler-SSManager-ParallelExecutor-146-0ad2f80f:15b1a06464c:23:1c]  INFO  RoutingClientBase.java (line 753) Retry on failure - dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list, starting attempt# 3 requestId 0ad2f80f:15b1a0657bd:ca:23, commandType REQUEST_BLOCK_REALLOCATION, to null
2017-03-29T13:01:39,975 [TaskScheduler-SSManager-ParallelExecutor-146-0ad2f80f:15b1a06464c:23:1c]  WARN  RoutingClientBase.java (line 640) Request failed, server null, key [CoS: urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690, type: SS, keyHash: 5219, level: 1], requestId 0ad2f80f:15b1a0657bd:ca:23, Process UNKNOWN, com.emc.storageos.data.object.exception.ObjectControllerException: dt urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: is in black list
2017-03-29T13:01:40,028 [SSConfigurationTracker-000]  INFO  SSPinger.java (line 70) Staring PING request to SS 10.210.248.15 with seq 1408
2017-03-29T13:01:40,029 [SSConfigurationTracker-000]  WARN  SSPinger.java (line 117) Partition:e9f8168a-c5a1-44d0-a8ea-adff1d7ea3f6 with state:SUSPECT has no block bins. SS 10.210.248.15, seq 1408
2017-03-29T13:01:40,029 [SSConfigurationTracker-000]  INFO  SSRecordHolder.java (line 146) Set size 0, SS 10.210.248.15
2017-03-29T13:01:40,029 [SSConfigurationTracker-000]  INFO  SSPinger.java (line 92) Completed PING request to SS 10.210.248.15 with seq 1408
2017-03-29T13:01:40,595 [client-heart-beat-monitor-000]  INFO  ClientHeartBeatImpl.java (line 66) start hb run
2017-03-29T13:01:41,961 [TaskScheduler-SSManager-ParallelExecutor-128-0ad2f80f:15b1a06464c:23:1c] ERROR  RemoteOperation.java (line 130) Wait for write completion timed out. RequestID 0ad2f80f:15b1a0657bd:dc:10d ServerIP 10.210.248.15:9091
2017-03-29T13:01:41,961 [TaskScheduler-SSManager-ParallelExecutor-128-0ad2f80f:15b1a06464c:23:1c]  WARN  RoutingClientBase.java (line 640) Request failed, server 10.210.248.15:9091, key [CoS: urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690, type: CT, keyHash: 823, level: 2], requestId 0ad2f80f:15b1a0657bd:dc:10d, Process CM, com.emc.storageos.data.object.exception.ObjectControllerException: wait for write completion timed out. RequestID 0ad2f80f:15b1a0657bd:dc:10d Server 10.210.248.15:9091
2017-03-29T13:01:41,963 [TaskScheduler-SSManager-ParallelExecutor-128-0ad2f80f:15b1a06464c:23:1c]  INFO  BlackList.java (line 46) added urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__CT_23_32_2: to dt black list ; expire 4219314197200 prev null
2017-03-29T13:01:42,916 [TaskScheduler-SSManager-ParallelExecutor-130-0ad2f80f:15b1a06464c:23:1c]  INFO  RoutingClientBase.java (line 753) Retry on failure - DT not initialized, request 0ad2f80f:15b1a0657bd:dd:1b, ip 10.210.248.15:1095, starting attempt# 10 requestId 0ad2f80f:15b1a0657bd:dd:1b, commandType REQUEST_BLOCK_REALLOCATION, to 10.210.248.15:1095
2017-03-29T13:01:42,916 [TaskScheduler-SSManager-ParallelExecutor-130-0ad2f80f:15b1a06464c:23:1c]  INFO  BlackList.java (line 59) removed urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: from dt black list ; expire 4213888103237 removeOrNot true nanoTime 4215267789846
2017-03-29T13:01:42,917 [TaskScheduler-SSManager-ParallelExecutor-130-0ad2f80f:15b1a06464c:23:1c]  INFO  BufferedFileWriterSharedChunkPrefetcher.java (line 570) try to trigger chunk creation for DT urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1:9bc29876-9185-4ee4-89f8-4eac722b1f74
2017-03-29T13:01:42,917 [TaskScheduler-SSManager-ParallelExecutor-130-0ad2f80f:15b1a06464c:23:1c]  WARN  AbstractRequestProcessor.java (line 217) failure processing remote request ID 0ad2f80f:15b1a0657bd:dd:1b com.emc.storageos.data.object.exception.ObjectControllerException: is readOnly, chunk or JR is not created yet, urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1:
2017-03-29T13:01:42,917 [TaskScheduler-SSManager-ParallelExecutor-130-0ad2f80f:15b1a06464c:23:1c]  INFO  AbstractRequestProcessor.java (line 305) Cannot utilize directory table for key hash 5219 due to error is readOnly, chunk or JR is not created yet, urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1:: request from 10.210.248.15 - 0ad2f80f:15b1a0657bd:dd:1b
2017-03-29T13:01:42,917 [TaskScheduler-SSManager-ParallelExecutor-130-0ad2f80f:15b1a06464c:23:1c]  WARN  RoutingClientBase.java (line 640) Request failed, server 10.210.248.15:1095, key [CoS: urn:storageos:VirtualArray:d0295c2a-3e1d-49b6-b6c4-67bbccf74690, type: SS, keyHash: 5219, level: 1], requestId 0ad2f80f:15b1a0657bd:dd:1b, Process SSM, com.emc.storageos.data.object.exception.ObjectControllerException: DT not initialized, request 0ad2f80f:15b1a0657bd:dd:1b, ip 10.210.248.15:1095
2017-03-29T13:01:42,918 [TaskScheduler-SSManager-ParallelExecutor-130-0ad2f80f:15b1a06464c:23:1c]  INFO  BlackList.java (line 46) added urn:storageos:OwnershipInfo:d0295c2a-3e1d-49b6-b6c4-67bbccf74690__SS_3_32_1: to dt black list ; expire 4220269184063 prev null
```

The output of `curl -X GET http://10.210.248.15:9101/stats/dt/DTInitStat` shows `<?xml version="1.0" encoding="UTF-8" standalone="yes"?><result><entry><total_dt_num>192</total_dt_num><unready_dt_num>0</unready_dt_num><unknown_dt_num>0</unknown_dt_num></entry></result>`

**Comment by padthaitofuhot** [comment]
This should be resolved in newest ECS 3.0.0 image emccorp/ecs-software-3.0.0; setup.py should be installed by default now. Let me know if you're still seeing this issue.

**Comment by padthaitofuhot** [comment]
Please reopen this issue if we can be of further assistance.  Thank you.
